R/nested-resampling.R

```diff
@@ -0,0 +1,122 @@
+library(tidymodels)
+library(bonsai)
+library(finetune)
+# pak::pak(c("tidymodels/finetune@nested"), ask = FALSE)
```

Comment:
The APIs here are experimental. The overall syntax might change so these results are pre-computed.